src/test/java/net/sf/jabref/importer/fileformat/PdfContentImporterTest.java

```diff
@@ -17,4 +31,19 @@ public void doesNotHandleEncryptedPdfs() throws IOException {
 
         assertEquals(Collections.emptyList(), result);
     }
-}
\ No newline at end of file
+
+    @Test
+    public void correctContent() throws IOException {
+        PdfContentImporter importer = new PdfContentImporter();
+        // The test folder contains pairs of PDFs and BibTeX files. We check each pair.
+        // Currently only one pair available
+        List<String> prefixes = Arrays.asList("LNCS-minimal");
```

Comment:
Like https://github.com/junit-team/junit/wiki/Parameterized-tests and see the FormatterTest for an example.